### DISCUSSION TITLE
问卷增加问题的建议

### DISCUSSION THREAD

**Discussion by jahn2007** [CollegesChat/university-information#266] [Enhancement]
想给问卷提一些建议，很希望明年再进行调查时可以添加如下问题
1.宿舍是否有空间安置台式机？台式机通常默认atx尺寸。能放几块显示屏？
2.考试难度，通过率，频率等
3.是否有rj45网口
4.是否允许自己拉宽带？

感谢❤️希望可以帮到明年的自己，同时也帮到更多的人

**Comment by thomeboy123** [comment]
建议增加学校有几个校区，然后回答的人在后面备注是哪些校区有哪些没有

**Comment by jahn2007** [comment]
又想起来几个新建议，感谢管理员维护问卷！
1.转专业政策如何？
2.先标明自己的校区、学习阶段。多数学校本科硕士博士的管理政策有异

**Comment by SYL0615** [comment]
建议增加一些提问
床有多大（0.9X1.9的还是什么）
可以装床帘吗
是不是要下载很多专门的APP（空调、饭卡、水卡缴费、校园跑、取快递、抢课之类的）大概要多少内存
